assets/js/hooks/emoji_picker.js

```diff
@@ -1,31 +1,33 @@
-import { createPicker } from "picmo";
+import { createPopup } from "@picmo/popup-picker";
 
 /**
  * A hook for the emoji picker input.
  */
 const EmojiPicker = {
   mounted() {
-    const rootElement = this.el.querySelector("[data-emoji-container]");
+    const button = this.el.querySelector("[data-emoji-button]");
     const preview = this.el.querySelector("[data-emoji-preview]");
     const input = this.el.querySelector("[data-emoji-input]");
-    const button = this.el.querySelector("[data-emoji-button]");
-
-    const pickerOptions = {
-      rootElement,
-      showSearch: false,
-      showPreview: false,
-    };
 
-    const picker = createPicker(pickerOptions);
+    const picker = createPopup(
+      {
+        showSearch: false,
```

Comment:
@aleDsz is there a specific reason we don't show the search bar? I feel like it is usually quite useful when you know what emoji you want to select.